hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -163,6 +164,21 @@ module top_${top["name"]} #(
   // this avoids lint errors
   assign unused_irq_id = irq_id;
 
+  // Alert list
+  prim_pkg::alert_tx_t [alert_pkg::NAlerts-1:0]  alert_tx;
+  prim_pkg::alert_rx_t [alert_pkg::NAlerts-1:0]  alert_rx;
+  // Escalation outputs
+  prim_pkg::esc_tx_t [alert_pkg::N_ESC_SEV-1:0]  esc_tx;
+  prim_pkg::esc_rx_t [alert_pkg::N_ESC_SEV-1:0]  esc_rx;
+
+% if not top["alert"]:
+  for (genvar k = 0; k < alert_pkg::NAlerts; k++) begin : gen_alert_tie_off
+    // tie off if no alerts present in the system
+    assign alert_tx[k].alert_p = 1'b0;
+    assign alert_tx[k].alert_n = 1'b1;
+  end
+% endif
```

Comment:
Do you think TOPs in OpenTitan always have alert signals? Will there be any case that no alert system instantiated?

Reply:
I think it is unlikely that we will not instantiate the alert handler. I just added this functionality in the template to make sure that no signals are left floating should we ever choose to omit it...